### ISSUE TITLE
Person template slightly broken

### ISSUE THREAD

**Issue by nielsdrost** [NLeSC/software.esciencecenter.nl#58]
The person page does not resolve "contactPersonOf" to sofware. Instead it displays the URL

Also, a list is displayed with incorrect layout (comma's).

**Comment by egpbos** [comment]
Indeed, the comma lists are very strange. This also affects the software layout, e.g. http://software.esciencecenter.nl/software/pidilib/. The second 'paragraph' is in fact an unordered list.

**Comment by wrvhage** [comment]
:)
They’re in fact lists of people, but we choose to not lay them out as a bullet list, because that takes too much space and looks bad. If you would change the code from a list to, e.g. spans or divs, then you lose the information that it’s a list. This is a common way to separate form and function on the Web. For instance, many website use lists to do their menu structures, but lay them out as menus.

Willem

| Dr. Willem van Hage | Technology Lead Data Management and Analytics | email: w.vanhage@esciencecenter.nl<mailto:w.vanhage@esciencecenter.nl> |
| Netherlands eScience Center | Science Park 140 | 1098 XG Amsterdam | The Netherlands | www.nlesc.nl<http://www.nlesc.nl/> |

On 13 Jun 2016, at 15:41, Patrick Bos <notifications@github.com<mailto:notifications@github.com>> wrote:


Indeed, the comma lists are very strange. This also affects the software layout, e.g. http://software.esciencecenter.nl/software/pidilib/. The second 'paragraph' is in fact an unordered list.

**Comment by egpbos** [comment]
Perhaps the css could be changed so that only lists of people get rendered like this, not all lists (see pidilib example).

**Comment by blootsvoets** [comment]
Yes, there should just be a `comma-list` class, and apply that class to lists of people or organizations in the metadata overview.
